examples/dao/scripts/run/withdraw_from_proposal.js

```diff
@@ -0,0 +1,45 @@
+const { tryExecuteTx } = require("./common/common.js");
+const { types } = require("@algo-builder/web");
+const { accounts } = require("./common/accounts.js");
+const { mkWithdrawFromProposalTx } = require("./common/tx-params.js");
+
+async function withdrawFromProposal(deployer, proposalLsig, proposer, voterA) {
+	const govTokenID = deployer.asa.get("gov-token").assetIndex;
+	console.log("gov token id: ", govTokenID);
+
+	// Needed to add the fund operation for it to work.
```

Comment:
Not sure about this issue. Maybe the way I run it is incorrect

Reply:
because the proposal must have tokens in the first place. In this proposal, you are basically sending gov tokens to the proposal. This is not what we want. You should remove `fundProposaltx`.
The flow is:
+ owner creates proposal
+ people vote (they firstly need to deposit vote tokens)
+ we execute the proposal (finalize it)
+ owner calls "clear_proposal" <- here the DAO will return deposit back to the proposal lsig.
+ and only then, an owner can call withdraw from the proposal lsig.

So basically this script should be called after voting and after finalizing and after `clear_proposal.js` - after that it  should work.

Reply:
The script was updated and the scenario works in the [dao-app-flow.js](https://github.com/scale-it/algo-builder/pull/621#diff-05aea0ea48d2b68d5f013c73e2277d2af1fe75bb4887582778a113381b68faf6). However when executing all the scripts manually I could not get the `execute` transaction to succeed. Not sure if im doing something incorrectly. Maybe we should ad a story?